fast/stages/0-bootstrap/automation.tf

```diff
@@ -127,7 +128,7 @@ module "automation-project" {
       }
     }
   )
-  services = concat(
+  services = setsubtract(concat(
```

Comment:
you can pass the unavailable services directly since your other PR was merged

Reply:
Done!